document/manuscript.md

```diff
@@ -213,7 +212,7 @@ and badges (also called shields) for the projects that can be included
 on your GitHub page, thereby making your projects easily identifiable
 as high quality and well-maintained.
 
-## Rule 7. Use and maintain your GitHub issue trackers
+## Rule 7. Use and maintain your issue trackers
```

Comment:
taking into account the suggestion of issue #58 we will make some of the titles more generic